### PR TITLE
[PFX-454] - fix broken jest tests in @godaddy/webapp apps

### PR DESCRIPTION
**Pull request by mmason2-godaddy**
## Summary
**First Error**
**Error:** Jest could not parse the `css` imports and the `@ux/pivot/style` import correctly.
**Fix**: [moduleNameMapper](https://github.com/godaddy/gasket/pull/678/files#diff-42c35ddd69ff03d916c4172420deb7f9da6c3077c179fc7d6f59c5a531c1a5e7R12) to parse imports correctly. Stubbed file [here](https://github.com/godaddy/gasket/pull/678/files#diff-f8958878b8d3dee758adb50cf1a1a33286ed4b9e46002dc8f216fd94d86fbd0fR1)
Screenshot 2024-02-15 at 4 15 48 PM

**Second Error**
**Error:** `[React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry.` For whatever reason `react-intl` wasn't working properly.
**Fix:** [Mock](https://github.com/godaddy/gasket/pull/678/files#diff-0dd0c4365a465bd16074b86ce5aa9d75db665e2b7a07ae889f650fed276b2f08R11) `react-intl`
Screenshot 2024-02-15 at 4 17 33 PM

**Third Error(Original error)**
**Error:** `Error: Not implemented: window.computedStyle(elt, pseudoElt)`
**Fix:** [Work around](https://github.com/godaddy/gasket/pull/678/files#diff-0dd0c4365a465bd16074b86ce5aa9d75db665e2b7a07ae889f650fed276b2f08R8) for this noisy error found via [this GH comment](https://github.com/nickcolley/jest-axe/issues/147#issuecomment-758804533)
Screenshot 2024-02-15 at 4 18 38 PM

## Changelog
- fix broken Jest tests in new Gasket apps

## Test Plan
New apps using `@godaddy/webapp` preset SHOULD have tests that pass. I'm having a hard time testing this locally with the preset.

### CODE REVIEW

File: packages/gasket-plugin-jest/generator/jest.config.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const customJestConfig = {
   collectCoverageFrom: ['**/*.js'],
   testEnvironmentOptions: {
     url: 'http://localhost/'
+  },
+  // Parse css imports & style paths
+  moduleNameMapper: {
+    '^.+\\.(css)$': '<rootDir>/test/style-mock.js',
+    '@ux/pivot/styles': '<rootDir>/test/style-mock.js'
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-jest/generator/jest.config.js:14]: Adding a style mock for css import is a good call.

However, the next mapper appears to be an app-specific import. This generated example jest config should not have it. If anything, add a comment or link to docs about customizing further.
```suggestion
```

**Review by agerard-godaddy — CHANGES_REQUESTED** [review]

**Comment by mmason2-godaddy** [comment]
So here's the PR that added the pivot styles -> https://github.com/gdcorp-uxp/gasket/pull/1338/files

Are we thinking I should just remove that and leave it broken?

File: packages/gasket-plugin-nextjs/generator/jest/test/index.spec.js [hunk 2]
```diff
@@ -3,11 +3,17 @@ import { render, screen } from '@testing-library/react';
 import { IndexPage } from '../pages/index';
 import { expect } from '@jest/globals';
 
+// Fix for noisy jsdom error
+// https://github.com/nickcolley/jest-axe/issues/147#issuecomment-758804533
+const { getComputedStyle } = window;
+window.getComputedStyle = (elt) => getComputedStyle(elt);
+
+jest.mock('react-intl');
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-nextjs/generator/jest/test/index.spec.js:11]: I understand this is a preemptive example mock - but does it blow up if `react-intl` is NOT installed for a particular app? If it still works, then I think it's fine to leave here, but add a comment about why it is in the example test for users to follow up on.

Comment (agerard-godaddy) [reply packages/gasket-plugin-nextjs/generator/jest/test/index.spec.js:12]: I was able to test it out and confirm it works.

**Review by agerard-godaddy — APPROVED** [review]

**Review by kbader-godaddy — APPROVED** [review]

**Pull request merged**